include/libopencm3/stm32/h7/pwr.h

```diff
@@ -0,0 +1,93 @@
+/** @defgroup pwr_defines PWR Defines
+
+@brief <b>Defined Constants and Types for the STM32H7xx Power Control</b>
+
+@ingroup STM32H7xx_defines
+
+@version 1.0.0
+
+@author @htmlonly &copy; @endhtmlonly 2019 Brian Viele <vielster@allocor.tech>
+
+@date 6 November 2019
+
+LGPL License Terms @ref lgpl_license
+ */
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2017 Matthew Lai <m@matthewlai.ca>
```

Comment:
You can't mix copyrights like this.  I have no opinion whose copyright it is, but it can't be both of you, with different years.